app_flutter/lib/service/appengine_cocoon.dart

```diff
@@ -26,22 +27,21 @@ class AppEngineCocoonService implements CocoonService {
   /// The Cocoon API endpoint to query
   ///
   /// This is the base for all API requests to cocoon
-  static const String _appengineAuthority = 'flutter-dashboard.appspot.com';
-  static const String _baseApiUrl = 'https://$_appengineAuthority/api';
+  static const String _baseApiUrl = 'https://flutter-dashboard.appspot.com';
 
   final http.Client _client;
 
   @override
   Future<CocoonResponse<List<CommitStatus>>> fetchCommitStatuses() async {
+    final String getStatusUrl = _apiEndpoint('/api/public/get-status');
+
     /// This endpoint returns JSON [List<Agent>, List<CommitStatus>]
-    final http.Response response =
-        await _client.get('$_baseApiUrl/public/get-status');
+    final http.Response response = await _client.get(getStatusUrl);
 
     if (response.statusCode != HttpStatus.ok) {
       print(response.body);
       return CocoonResponse<List<CommitStatus>>()
-        ..error =
-            '$_baseApiUrl/public/get-status returned ${response.statusCode}';
+        ..error = '/api/public/get-status returned ${response.statusCode}';
```

Comment:
Why this line doesn't call _apiEndpoint?

Reply:
It's just an error message, so I think just giving the endpoint is fine. I was trying to keep the error messages concise so they could be shown in the UI.